src/routes/LeaveMessage/component.js

```diff
@@ -183,10 +183,10 @@ export default class LeaveMessage extends Component {
 			{...props}
 		>
 			<Screen.Content>
-				<p className={createClassName(styles, 'leave-message__main-message')}>
-					{hasForm ? message || defaultMessage : unavailableMessage || defaultUnavailableMessage}
-				</p>
-
+				<p className={createClassName(styles, 'leave-message__main-message')}
+					// eslint-disable-next-line react/no-danger
+					dangerouslySetInnerHTML={{ __html: hasForm ? message || defaultMessage : unavailableMessage || defaultUnavailableMessage }}
```

Comment:
Since the idea is to enable the admins to use links and rich content on the livechat message, wouldn't it be better to use markdown instead? We do already have a render function here [here](https://github.com/RocketChat/Rocket.Chat.Livechat/blob/develop/src/components/Messages/MessageText/markdown.js#L84-L85)

I think using dangerouslySetInnerHTML should be reserved to extreme cases 🙂

Reply:
Hi Martin, Nice idea :ok_hand: I've made the changes to support only markdown format. Let me know if it looks good now :smile: Thanks!!